src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = 40;
+  const threeDaysDiscount = 20;
+  const threeDays = 3;
+  const sevenDaysDiscount = 50;
+  const sevenDays = 7;
+
+  if (days >= sevenDays) {
+    return basePrice * days - sevenDaysDiscount;
+  } else if (days >= threeDays) {
+    return basePrice * days - threeDaysDiscount;
+  } else {
+    return basePrice * days;
+  }
```

Comment:
You should avoid using `else` after `if` blocks that contain a `return` statement. The code after the `if` block won't be executed if the condition is met, so `else` is unnecessary and can be removed to simplify the code structure.